src/sentry/monitors/validators.py

```diff
@@ -223,7 +224,12 @@ class MonitorCheckInValidator(serializers.Serializer):
             ("in_progress", CheckInStatus.IN_PROGRESS),
         )
     )
-    duration = EmptyIntegerField(required=False, allow_null=True)
+    duration = EmptyIntegerField(
+        required=False,
+        allow_null=True,
+        max_value=BoundedPositiveIntegerField.MAX_VALUE,
```

Comment:
Do we do this type of thing anywhere else?

Reply:
A couple places but we usually just type out `2147483647` and stick it in the file

Reply:
I guess since this is a static number probably OK to copy instead of reference

Reply:
Nah I'm fine with the constant, was just curious if this was our usual pattern